mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule JokenJwks.MixProject do
       {:joken, "~> 2.6"},
       {:jason, "~> 1.4"},
       {:tesla, "~> 1.4"},
-      {:hackney, "~> 1.17.4 or ~> 1.18.0", optional: true},
+      {:hackney, "~> 1.17.4 or ~> 1.18.0 or ~> 1.19.0 or ~> 1.20.0", optional: true},
```

Comment:
I believe we can make it like:

```suggestion
      {:hackney, "~> 1.18", optional: true},
```

Which would allow all of the above except 1.17. I can´t quite recall why I chose this particular setup currently...

Reply:
Yes, that's a good idea. I created a separate issue #57 to propose this, as I was unsure about the reasons. Great that we can resolve that, too!